real build fail?

```
======================================================================
FAIL: test_multiple_batches_restore (casexml.apps.phone.tests.test_batched_mode.BatchRestoreTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_batched_mode.py", line 48, in test_multiple_batches_restore
    self.assertEqual(restore_config.num_batches, 4)
AssertionError: None != 4
```

--- a/corehq/toggles.py
+++ b/corehq/toggles.py
@@ -56,7 +56,7 @@ class PredicatablyRandomToggle(StaticToggle):
     """
 
     def __init__(self, slug, label, namespace, randomness):
-        super(PredicatablyRandomToggle, self).__init__(slug, label, [namespace])
+        super(PredicatablyRandomToggle, self).__init__(slug, label, list(namespace))
         assert namespace, 'namespace must be defined!'
         self.namespace = namespace
         assert 0 <= randomness <= 1, 'randomness must be between 0 and 1!'

